### PR TITLE
Fixes #20625 - Add upgrade scenarios

### PR DESCRIPTION
**Pull request by swapab**
~pre-requisite : #67~

Upgrade scenarios for Satellite versions:
* 6.2 -> 6.2.z
* 6.2 -> 6.3
* 6.3 -> 6.3.z

### CODE REVIEW

**Comment by iNecas** [comment]
Let's wait with this until https://github.com/theforeman/foreman_maintain/pull/67 is in, as it adds additional logic into the upgrades

**Comment by iNecas** [comment]
#67 merged

**Comment by swapab** [comment]
@iNecas This PR is ready for review.

**Comment by iNecas** [comment]
While on that, let's add also 6.3.z scenario: might be useful for upgrades between snap + we will need to have it anyway.

**Comment by iNecas** [comment]
Otherwise, looks good

**Comment by swapab** [comment]
@iNecas Added 6.3.z scenario.

File: definitions/scenarios/upgrade_to_satellite_6_3.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    upgrade_metadata do
+      description 'migration scripts to Satellite 6.3'
+      tags :migrations
+    end
+
+    def compose
+      add_step(Procedures::Repositories::Setup.new(:version => '6.3'))
+      add_step(Procedures::Packages::Update.new(:assumeyes => false))
+      add_step(Procedures::Installer::Upgrade.new)
+      add_steps(find_procedures(:label => :installer_upgrade))
```
Comment (iNecas) [inline definitions/scenarios/upgrade_to_satellite_6_3.rb]: Please note that this is actually a duplicate line (my mistake). I'm fixing it in the original code here https://github.com/theforeman/foreman_maintain/pull/88/files#diff-b8fa89d315b87599a0ce9bd5f2bffda3L48 but let's not copy the mistake over other places

Comment (swapab) [inline definitions/scenarios/upgrade_to_satellite_6_3.rb]: Deleted the duplicate lines. From newly added scenarios.

**Comment by iNecas** [comment]
Thanks @swapab

**Pull request merged**